kubernetes-model/kubernetes-model-batch/pom.xml

```diff
@@ -0,0 +1,235 @@
+    <artifactId>kubernetes-model-generator</artifactId>
+    <version>4.9-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>kubernetes-model-batch</artifactId>
+  <packaging>bundle</packaging>
+  <name>Fabric8 :: Kubernetes Model :: Batch</name>
+
+  <properties>
+    <clone-kube>true</clone-kube>
+  </properties>
+
+  <dependencies>
+    <dependency>
+      <groupId>io.fabric8</groupId>
+      <artifactId>kubernetes-model-common</artifactId>
+      <version>${project.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>io.fabric8</groupId>
+      <artifactId>kubernetes-model-core</artifactId>
+      <version>${project.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>com.fasterxml.jackson.module</groupId>
+      <artifactId>jackson-module-jaxb-annotations</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.junit.jupiter</groupId>
+      <artifactId>junit-jupiter-engine</artifactId>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.junit.jupiter</groupId>
+      <artifactId>junit-jupiter-migrationsupport</artifactId>
+      <scope>test</scope>
+    </dependency>
+
+    <dependency>
+      <groupId>net.javacrumbs.json-unit</groupId>
+      <artifactId>json-unit-fluent</artifactId>
+      <version>2.7.0</version>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.projectlombok</groupId>
+      <artifactId>lombok</artifactId>
+      <scope>provided</scope>
+    </dependency>
+  </dependencies>
+
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.jsonschema2pojo</groupId>
+        <artifactId>jsonschema2pojo-maven-plugin</artifactId>
+        <version>${jsonschema2pojo.version}</version>
+        <configuration>
+          <sourcePaths>
+            <sourcePath>${project.basedir}/src/main/resources/schema/kube-schema.json</sourcePath>
+          </sourcePaths>
+          <targetPackage>io.fabric8.kubernetes.api.model</targetPackage>
+          <includeConstructors>true</includeConstructors>
+          <includeJsr303Annotations>false</includeJsr303Annotations>
+          <includeToString>false</includeToString>
+          <includeHashcodeAndEquals>false</includeHashcodeAndEquals>
+          <outputDirectory>${project.build.directory}/generated-sources</outputDirectory>
+          <customAnnotator>io.fabric8.kubernetes.annotator.KubernetesTypeAnnotator</customAnnotator>
+          <annotationStyle>none</annotationStyle>
+        </configuration>
+        <executions>
+          <execution>
+            <goals>
+              <goal>generate</goal>
+            </goals>
+          </execution>
+        </executions>
+        <dependencies>
+          <dependency>
+            <groupId>io.fabric8</groupId>
+            <artifactId>kubernetes-model-annotator</artifactId>
+            <version>${project.version}</version>
+          </dependency>
+        </dependencies>
+      </plugin>
```

Comment:
Make this pom consistent with the others (remove profile + leave just minimal configuration entry for the plugin definition)